homeassistant/components/assist_satellite/__init__.py

```diff
@@ -86,6 +96,34 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         "async_internal_start_conversation",
         [AssistSatelliteEntityFeature.START_CONVERSATION],
     )
+    component.async_register_entity_service(
+        "ask_question",
+        vol.All(
+            cv.make_entity_service_schema(
+                {
+                    vol.Optional("question"): str,
+                    vol.Optional("question_media_id"): str,
+                    vol.Optional("preannounce"): bool,
+                    vol.Optional("preannounce_media_id"): str,
+                    vol.Optional("answers"): [
```

Comment:
This should be required?

Reply:
No because without answers you can just get back the raw text without any parsing. Very handy if you want to do your own.